### DISCUSSION TITLE
[Projects Beta] Bulk archive selected issues

### DISCUSSION THREAD

**Discussion by amacneil** [community/community#5846] [Projects and Issues] [answered]
In issues beta, I have the ability to select multiple issues at once.

But, I don't see any way to mass-archive selected issues, nor select or archive a column of issues.

Typically, once per week I want to archive all the issues in the "done" column, which is currently very painful because I have to do it one at a time (and every archive action requires a confirmation modal).

**Comment by patrickklaeren** [comment, marked as answer]
@amacneil You can do this by selecting and holding shift, on the left side column, if in a list view and pressing `e` on your keyboard.

**Reply by amacneil** [reply]
Thank you. That's strange, I definitely tried this yesterday and it did not work, but today I tried and it did. I will test more thoroughly next time I have issues that need archiving and see if I can reproduce it.

Either way - I think functionality shouldn't _only_ be accessible under a keyboard shortcut, so a way to do this in the UI is needed. Also, a "select entire column" function would be very helpful.

**Reply by patrickklaeren** [reply]
It's not super intuitive. There's a lot bulk editing features missing right now.

**Reply by azenMatt** [reply]
We _just_ release the archive functionality so you might have just missed it! 

You can also use the item menu to archive - but right now we have a bug that is being fixed that dismisses your selection on the board when you try this. But that will be fixed shortly. 

You can see some more details on the recent release here https://github.blog/changelog/2021-09-29-the-new-github-issues-09-29-update/

**Reply by azenMatt** [reply]
@patrickklaeren - more bulk updates are coming soon - are you specifically thinking of updating multiple cell values? Or do you have something else in mind?

**Reply by patrickklaeren** [reply]
@azenMatt Definitely multiple select moving up and down, multi cell editing is what I'm thinking. I've left separate discussion items for this on the feedback board.

Other than that, this is a very good product. Good job to the team.
